tsconfig.json

```diff
@@ -14,7 +14,8 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react"
+    "jsx": "react",
+    "noFallthroughCasesInSwitch": true
```

Comment:
This was required due to the issue described here: https://github.com/facebook/create-react-app/issues/9429#issuecomment-674994709